client/src/components/layouts/Default.js

```diff
@@ -5,6 +5,7 @@ import { bindActionCreators } from 'redux';
 import { createSelector } from 'reselect';
 import Helmet from 'react-helmet';
 import fontawesome from '@fortawesome/fontawesome';
+import { Grid, Col, Row } from '@freecodecamp/react-bootstrap';
```

Comment:
Where is it used?

Reply:
thanks for noticing it. @ValeraS  I removed that line.